x-pack/test_serverless/README.md

```diff
@@ -212,9 +212,23 @@ defining and authenticating with custom roles in both UI functional tests and AP
 
 To test role management within the Observability project, you can execute the tests using the existing [config.feature_flags.ts](x-pack/test_serverless/functional/test_suites/observability/config.feature_flags.ts), where this functionality is explicitly enabled. Though the config is not run on MKI, it provides the ability to test custom roles in Kibana CI before the functionality is enabled in MKI. When roles management is enabled on MKI, these tests can be migrated to the regular FTR config and will be run on MKI.
 
-For compatibility with MKI, the role name `customRole` is reserved for use in tests. The test user is automatically assigned to this role, but before logging in via the browser, generating a cookie header, or creating an API key in each test suite, the role’s privileges must be updated.
+When running tests locally against MKI, ensure that the `.ftr/user_roles.json` file includes the reserved role name `custom_role_worker_1` along with its credentials. This role name has been updated for compatibility with Scout, which supports parallel test execution and allows multiple credential pairs to be passed.
```

Comment:
Is the file be named `.ftr/role_users.json` or `.ftr/user_roles.json`? From a quick search in the Kibana project and a quick look at my own `.ftr` directory, it's probably `.ftr/role_users.json`.

This is what the [test_serverless README](https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/README.md#run-tests-on-mki) mentions:

> Create `.ftr/role_users.json` in the root of Kibana repo.

Reply:
I see this was fixed in [16a2516](https://github.com/elastic/kibana/pull/217882/commits/16a25162afe82bc4b6fc3544f4297e409d42964b), thanks!